pypsa/plot.py

```diff
@@ -308,40 +334,40 @@ def draw_map(network=None, jitter=None, ax=None, boundaries=None,
 
     #First choice should be cartopy
     if cartopy_present:
-        resolution = '50m' if isinstance(drawmap, bool) else drawmap
+        resolution = '50m' if isinstance(geomap, bool) else geomap
         assert resolution in ['10m', '50m', '110m'], (
                 "Resolution has to be one of '10m', '50m', '110m'")
-        geomap = ax.projection
+        gmap = ax.projection
+        orig_projection = get_projection_from_crs(network.srid)
         transformed = pd.DataFrame(
-                    geomap.transform_points(
-                        ccrs.PlateCarree(), x.values, y.values),
+                    gmap.transform_points(
+                        orig_projection, x.values, y.values),
                         columns=['x', 'y', 'z'], index=network.buses.index)
         x, y = transformed.x, transformed.y
-        ax.set_extent([x1, x2, y1, y2], crs=ccrs.PlateCarree())
+        ax.set_extent([x1, x2, y1, y2], crs=orig_projection)
         ax.coastlines(linewidth=0.4, zorder=-1, resolution=resolution)
         border = cartopy.feature.BORDERS.with_scale(resolution)
         ax.add_feature(border, linewidth=0.3)
 
     elif basemap_present:
-        resolution = 'l' if isinstance(drawmap, bool) else drawmap
-        geomap = Basemap(resolution=resolution, epsg=network.srid,
+        resolution = 'l' if isinstance(geomap, bool) else geomap
+        gmap = Basemap(resolution=resolution, epsg=network.srid,
                        llcrnrlat=y1, urcrnrlat=y2, llcrnrlon=x1,
                        urcrnrlon=x2, ax=ax)
-        geomap.drawcountries(linewidth=0.3, zorder=-1)
-        geomap.drawcoastlines(linewidth=0.4, zorder=-1)
+        gmap.drawcountries(linewidth=0.3, zorder=-1)
+        gmap.drawcoastlines(linewidth=0.4, zorder=-1)
 
-        x, y = geomap(x.values, y.values)
+        x, y = gmap(x.values, y.values)
         x = pd.Series(x, network.buses.index)
         y = pd.Series(y, network.buses.index)
```

Comment:
Well, the `basemap` bit is as broken, as it has been before, but I can understand that you don't want to fix it. You can leave it if you want.

Reply:
I found that setting the bounds of `Basemap` (`llcrnrlat`, etc.) dramatically changes the projection. Since the bounds are not given by the user, but somewhat arbitrary, this basemap should not be used for conversion. (I think raising an error when basemap is used in combination with a non-standard projection might be safest.)